### PR TITLE
feat: :sparkles: add new columns

### PR DESCRIPTION
**Pull request by MagnusGjosund**

### CODE REVIEW

**Comment by MagnusGjosund** [comment]
deploy test

File: libs/heattraceapp/src/lib/config/tableConfig.tsx [hunk 1]
```diff
@@ -94,12 +123,44 @@ const columnDefinitions: [ColDef<HeatTrace>, ...ColDef<HeatTrace>[]] = [
   },
   { headerName: domainNames.currentStep, valueGetter: (pkg) => pkg.data?.checklistStep },
   {
+    colId: 'RfCPlannedForecastDate',
     headerName: 'RFC',
```
Comment (Gustav-Eikaas) [inline libs/heattraceapp/src/lib/config/tableConfig.tsx:97]: Is rfc colId misspelled?

File: libs/heattraceapp/src/lib/config/tableConfig.tsx [hunk 2]
```diff
@@ -94,12 +123,44 @@ const columnDefinitions: [ColDef<HeatTrace>, ...ColDef<HeatTrace>[]] = [
...
+    //   if (!props.data?.mechanicalCompletionHandoverStatus) return null;
+    //   return (
+    //     <StatusCircle
+    //       content={props.data.mechanicalCompletionHandoverStatus}
+    //       statusColor={
+    //         statusColorMap[props.data.mechanicalCompletionHandoverStatus as BaseStatus]
+    //       }
+    //     />
+    //   );
+    // },
```
Comment (Gustav-Eikaas) [inline libs/heattraceapp/src/lib/config/tableConfig.tsx:152]: Why is this commented out?

File: libs/pipingapp/src/lib/config/tableConfig.tsx [hunk 3]
```diff
@@ -86,7 +86,7 @@ const columnDefinitions: [ColDef<Pipetest>, ...ColDef<Pipetest>[]] = [
   },
   {
     headerName: domainNames.mcStatus,
-    colId: 'MechanicalCompletionStatus',
```
Comment (Gustav-Eikaas) [inline libs/pipingapp/src/lib/config/tableConfig.tsx:89]: Are you sure this lowercase change is correct?

**Review by Gustav-Eikaas — APPROVED** [review]
Looks good, please resolve all comments before merging!

File: libs/pipingapp/src/lib/config/tableConfig.tsx (hunk 3 above)
Comment (MagnusGjosund) [reply libs/pipingapp/src/lib/config/tableConfig.tsx:89]: no broken windows here

**Pull request merged**